apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -308,15 +281,6 @@ defmodule StateMediator.Integration.GtfsTest do
     end
   end
 
-  describe "schedules" do
-    test "CR-Fairmount has alternate route trips" do
```

Comment:
The new schedule no longer has an Franklin Line trips which operate via the Fairmount Line.